src/connectors/geo-search/connectGeoSearch.js

```diff
@@ -251,7 +254,7 @@ const connectGeoSearch = (renderFn, unmountFn) => {
       widgetState.lastRefinePosition = state.aroundLatLng || '';
       widgetState.lastRefineBoundingBox = state.insideBoundingBox || '';
 
-      widgetState.internalToggleRefineOnMapMove = createInternalToggleRefinementonMapMove(
+      widgetState.internalToggleRefineOnMapMove = createInternalToggleFacetRefinementOnMapMove(
```

Comment:
To revert (see https://github.com/algolia/instantsearch.js/pull/3207#discussion_r226257220).